data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/migrated/schema/BeaconStateBellatrix.json

```diff
@@ -106,14 +106,18 @@
       "type" : "array",
       "items" : {
         "type" : "string",
-        "format" : "byte"
+        "example" : "1",
+        "description" : "unsigned 8 bit integer, max value 255",
+        "format" : "uint8"
```

Comment:
Not strictly supported but to be fair neither is byte so probably not a big issue

Reply:
It matches `uint64` :)

Reply:
yeh basically we're mis-using the format but thats fine hopefully